### DISCUSSION TITLE
How to deploy a website to Production without gulp

### DISCUSSION THREAD

**Discussion by domimueller** [holger1411/understrap#1511] [Q&A]
Hi there
I justed developed a new wordpress site using understrap and want to go live. Now, the question is how to generate dist folder and activate production mode without gulp tasks? -> error message No gulpfile found... 
In the Changelog, I found an entry that they want to go away from gulp.. which deployment process should we follow? The Documentation of understrap is still talkting about gulp..

Changelog: https://github.com/understrap/understrap/blob/main/.github/CHANGELOG.md
Understrap Documentaiton: https://usdocu.tarthemes.com/docs/0.8.2/development/developing-with-npm-and-gulp/

thank you in advance for any help.
Kind regards
 domimueller

**Comment by bacoords** [comment]
Hi, 

That function will be returning in the next release as `npm run dist-build` and `npm run dist-clean` (https://github.com/understrap/understrap/pull/1432)

**Reply by domimueller** [reply]
Hey thanks for the response!
So meanwhile you recommend just to upload the theme without node_modules and src folder. Right? 

Thank you very much

**Reply by bacoords** [reply]
Yes that's correct. We remove node_modules, but we often keep the /src/ folder because it provides anything you or another dev might need to build from in the future, but you may not want it.
